docs/tools/vdb_table/data/lancedb.json

```diff
@@ -148,6 +148,15 @@
     "source_url": "",
     "comment": ""
   },
+  "index_types": {
+    "value": [
+      "FLAT",
+      "IVF",
```

Comment:
I would remove IVF as the defaults in LanceDB always include a PQ component to help with recall.